def throw_error():
    raise RuntimeError
